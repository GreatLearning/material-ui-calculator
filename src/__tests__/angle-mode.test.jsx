import renderer from 'react-test-renderer';
import AngleMode from '../angle-mode';
import React from 'react';

it('renders correctly', () => {
  const tree = renderer
    .create(<AngleMode angleMode="deg" />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
