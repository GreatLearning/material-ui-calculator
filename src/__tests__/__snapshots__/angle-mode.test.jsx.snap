// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className={undefined}
>
  <div
    className="Component-mode-1 Component-active-2"
    onClick={[Function]}
  >
    deg
  </div>
  <div
    className="Component-mode-1"
    onClick={[Function]}
  >
    rad
  </div>
</div>
`;
